File: PHP/card.php
<?php
class card extends Payments{
    public $cvv;
    public $dateCard;
    public $numberCard;
  
    
    public function __construct($id, $cvv, $dateCard, $numberCard){
        parent::__construct($id);
        $this->cvv = $cvv;
        $this->dateCard = $dateCard;
        $this->numberCard = $numberCard;
    }
    
}
?>

File: Python/card.py
from Paymenta import Payments

class card(Payments):
    cvv = int
    dateCard = int
    numberCard = int
    
    def __init__(self, id , cvv, dateCard, numberCard):
        super().__init__(id)
        self.cvv = cvv
        self.dateCard = dateCard
        self.numberCard = numberCard